crates/bdk/tests/wallet.rs

```diff
@@ -135,8 +135,8 @@ fn test_get_funded_wallet_tx_fee_rate() {
     // sats are the transaction fee.
 
     // tx weight = 452 bytes, as vbytes = (452+3)/4 = 113
-    // fee rate (sats per vbyte) = fee / vbytes = 1000 / 113 = 8.8495575221 rounded to 8.849558
-    assert_eq!(tx_fee_rate.as_sat_per_vb(), 8.849558);
+    // fee rate (sats per vbyte) = fee / vbytes = 1000 / 113 = 8.8495575221 rounded up to 9
+    assert_eq!(tx_fee_rate.to_sat_per_vb_ceil(), 9);
```

Comment:
since we lose precision with `to_sat_per_vb_ceil`, we should assert the value in sat/kwu also.